.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -23,3 +34,5 @@
 
 #### 5. Any other information you think might be useful for us to know?
 
+
+#### 6. Please tell us your browser version.
```

Comment:
Browser version is in the debug info, which we already request, as a part of the browser's User Agent string.

Reply:
Okay. That's fixed.